agent/php_error.c

```diff
@@ -580,6 +696,13 @@ nr_status_t nr_php_error_record_exception(nrtxn_t* txn,
     error_message = nr_formatf("%s'%s'", prefix, klass);
   }
 
+  /*
+   * Error Fingerprinting Callback
+   */
+  if (NULL != NRPRG(error_group_user_callback)) {
+    nr_php_error_fingerprint_callback(txn, klass, message, file, stack_json);
```

Comment:
Somehow I feel a verb is missing from this function name. This function is not a callback, it is supposed to call user's callback, right? If that's the case then maybe `nr_php_error_call_error_group_callback` or `nr_php_error_set_group` would be better?

Reply:
It's a balance between being descriptive without being overly verbose- How about `nr_php_error_call_fingerprint_callback` or `nr_php_call_error_fingerprint_callback`? 
I think we have to maintain some reference to the fact that this function is calling a callback and not just us setting a value.

Reply:
It might be better (for searching purposes) to consistently name things pertaining to the same functionality. Some places use `error_group_user_callback`, others use `fingerprint_callback` and others use `error_group_callback`. Could we make it consistent throughout the codebase and use `error_group_callback` (after the API)? And if so, the function would be named `nr_php_error_call_error_group_callback`.

Reply:
Yeah, I can agree with that- changed to `nr_php_error_call_error_group_callback ` in 8d06bbc.